Manual updates with some more pictures and a cleaner look (#111)
vegastrike/Assets-Masters: VSPlayersGuide.tex

\begin{tabular}{|p{0.4in}|p{0.3in}|p{4.1in}|} \hline 
\textbf{ Name } & \textbf{Key } & \textbf{Description } \\ \hline 

> View Forward  & \key{1} & This view is what appears initially in flight and is otherwise known as the cockpit view. This view contains all the necessary information and displays to fly. While in view forward, pressing \key{F1} will change from the cockpit view (with cockpit struts) to the Heads Up Display (HUD). The HUD provides the same information as the forward view but removes the cockpit surround. Press the key once more to return to cockpit view.  \\ \hline 

Reviewer: The additional mention of the F1 key needs to be removed. F1 now means "send a friendly message to the current target." I'm not sure if pressing 1 more than once would switch to HUD view. Perhaps so, perhaps not. I guess I should try it and find out.

Author: Thank you for the thorough review, I did test it and no such thing happened, thus I removed the wording around it.

\begin{tabular}{|p{0.6in}|p{0.4in}|p{3.9in}|} \hline 
\textbf{ Name } & \textbf{Key } & \textbf{Description } \\ \hline 

> Chasecam Panning Camera  & \key{F5} \key{F6} & This view seats a camera behind and above your ship's engines, looking forward. This position is static no matter the ship's movements. Best imagined as having a second ship tailing your current ship, with you in that second cockpit. This view allows you to examine all markings and detail on your ship. This view positions a camera at a single point in space, facing your ship. You may rotate the camera about your ship using the panning keys.  \\ \hline 

Reviewer: This text is going to need some more work. F5 and F6 bring up two different views. I have attempted to describe them below. See if my descriptions make sense.

```suggestion
Chasecam & \key{F5} & This view seats a camera behind and above your ship's engines, looking forward. This camera stays behind and above your ship's engines looking forward, no matter the ship's movements. This view allows you to examine all markings and detail on your ship.  \\ \hline 
Panning Camera & \key{F6} & Best imagined as having a second ship tailing your current ship, with you in that second cockpit. This view positions a camera at a single point in space, facing your ship. This camera does not move when your ship yaws, rolls, or pitches, so it allows you to examine all markings and detail on your ship from various angles. You may rotate the camera about your ship using the panning keys.  \\ \hline 
```

> Target Camera Target Panning Camera  & \key{F7} \key{F8} & This view places the camera at a location looking past your ship towards your current target. This view positions a camera at a single point in space, facing your target. You may rotate the camera about your target using the panning keys.  \\ \hline 

Reviewer: ```suggestion
Target Camera & \key{F7} & This view positions a camera at a single point in space, facing your target. You may rotate the camera about your target using the panning keys.  \\ \hline 
Random Camera & \key{F8} & This view positions a camera looking at either your ship or your target, from a random angle.  \\ \hline 
```

> Zoom View In Zoom View Out  & \key{F11} \key{F12} & While in one of the camera views, this function will move the viewpoint towards the object viewed. While in one of the camera views, this function will move the viewpoint towards the object viewed.  \\ \hline 

Reviewer: ```suggestion
Zoom View In & \key{F9} & While in one of the camera views, this function will move the viewpoint towards the object viewed.  \\ \hline 
Zoom View Out & \key{F10} & While in one of the camera views, this function will move the viewpoint away from the object viewed.  \\ \hline 
```
